Art/anorme-loading/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Animation Nation challenge</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
# Incorrect CSS filesheet 
After changing the file name you need to adjust the link href